index.html

```diff
@@ -8,20 +8,33 @@
     </head>
     <body class="overflow-hidden">
         <header>
-            <nav class="flex w-full h-16 bg-black text-white items-center">
+            <nav class="flex w-full h-[8vh] bg-black text-white items-center bg-gradient-to-t from-gray-800 via-gray-900 to-black">
                 <a href="#" class="flex items-center justify-start"><img src="assets/logo.png" alt="A.K.A" width="64" height="100"></a>
                 <a href="#" class="grid ml-6">Home</a>
                 <a href="#" class="grid ml-6">About</a>
                 <a href="#" class="grid ml-6">Content</a>
             </nav>
         </header>
         <main class="bg-hero bg-cover">
-            <div class="flex h-screen flex-col items-center justify-center">
+            <div class="flex min-h-[92vh] flex-col items-center justify-center">
                 <!--Title of the page-->
-                <h1 class="my-4 font-bold text-8xl tracking-tight bg-gradient-to-r from-green-500 via-blue-500 to-pink-500 bg-clip-text text-transparent leading-normal">
+                <h1 class="my-2 font-bold text-[8vw] tracking-tight bg-gradient-to-r from-green-500 via-blue-500 to-pink-500 bg-clip-text text-transparent leading-normal">
```

Comment:
as stated above, I believe instead of having the text size be based on the width of the screen, it would be best to keep it as `8xl` or `7xl` and instead add a tag for when the screen is not mobile to resize the title such as `md:text-7xl text-4xl` which would use `text-7xl` for anything larger than `md` which is by default `768px`

Reply:
Made a commit bbbbb7d2e0a8194acb7b147bb3725d98a0e2d2d5.